ajax/libs/markerclustererplus/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "markerclustererplus",
+  "filename": "markerclusterer.min.js",
+  "version": "2.1.4",
+  "description": "The library creates and manages per-zoom-level clusters for large amounts of markers.",
+  "keywords": [
+    "google",
+    "maps",
+    "markerclustererplus"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mahnunchik/markerclustererplus.git"
+  },
+  "author": {
+    "name": "mahnunchik",
+    "url": "https://github.com/mahnunchik"
+  },
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mahnunchik/markerclustererplus.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "markerclusterer.js",
+          "markerclusterer_packaged.js"
```

Comment:
@pvnr0082t Should we only fetch `src/markerclusterer.js` in auto-update config, but manually add the `markerclusterer_packed.js` for the old version as maruilian11 suggest?